### ISSUE TITLE
Support Google Fonts Icons

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#1505]
Should we support these? https://fonts.google.com/icons
Also, is it feasible?

**Comment by wxwxwxwx9** [comment]
I think it's possible. Maybe we can explore implementing it as a plugin?

**Comment by damithc** [comment]
> I think it's possible. Maybe we can explore implementing it as a plugin?

Support for other icon types are built-in. We can implement this as a plugin only if there is a good reason to differ from others. e.g., if adding this impose a significant performance cost, we can provide it as an optional feature (i.e., plugin)

**Comment by riyadh-h** [comment]
Just checking if this issue is still active. Thank you.

**Comment by damithc** [comment]
@riyadh-h feel free to give it a try

**Comment by riyadh-h** [comment]
So, I tried using these icons, and they don't seem to be aligned with any surrounding text:

![google-material-icons](https://user-images.githubusercontent.com/69278589/130746982-1d5bd67e-4041-44ef-be85-6be5bd0dfecc.png)

I assumed that the font size used for the icons was larger compared to the text's, so I tried resizing them to 18px.
However, this only worsened the render quality of the icons in my opinion:

![google-material-icons-small](https://user-images.githubusercontent.com/69278589/130747093-23d8c395-8617-4d85-8e13-d09d4697fea3.png)

Do you think we should proceed with the original size, or is there another way on how to go about this?

**Comment by tlylt** [comment]
Just curious... have you tried to align the icon via some CSS?

**Comment by riyadh-h** [comment]
@tlylt Not really, but now that you've mentioned it, I tried giving a `vertical-align` property that's set to `bottom`, and it looks perfect!

![google-material-icons-works](https://user-images.githubusercontent.com/69278589/130750190-afebb7a5-5c8f-4002-8c61-5ed5259a103b.png)

Cheers 😄
